doc: fix typo in doc/repl.md line: 6

utano320 opened the pull request:
<!--
Thank you for your pull request. Please review below requirements.

Bug fixes and new features should include tests and possibly benchmarks.

Contributors guide: https://github.com/nodejs/node/blob/master/CONTRIBUTING.md
-->

##### Checklist
<!-- Remove items that do not apply. For completed items, change [ ] to [x]. -->

- [x] documentation is changed or added
- [x] commit message follows commit guidelines

##### Affected core subsystem(s)
<!-- Provide affected core subsystem(s) (like doc, cluster, crypto, etc). -->


##### Description of change
<!-- Provide a description of the change below this comment. -->

fix doc/api/repl.md line 6 "includable" => "inclidable"

yosuke-furukawa commented:
Ooops, you introduce other typo... includible is correct word?

utano320 commented:
Oh, I'm sorry! I fixed it again ...

shigeki commented:
@utano320 I would like to confirm if your git name is correctly set before landing. If you want to prefer to using your real name such as your profile in github, please change it.  And when you change the commit name, please squash two commits into one.

lpinca commented:
I'm not a native speaker but I think "includable" is also correct.

utano320 commented:
@shigeki I changed the names of Author and Committer to real names and squashed the two commits to one.

@lpinca Thanks! It certainly seems like that... "includable" was described as "Alternative form of includible" in dictionary.

shigeki commented:
Thanks. Landed in 30bf123e41b3a4027aaec37464b471c01dcb2d94.
Congrats on your first contribution to Node.
